web-ui/src/main/resources/catalog/locales/en-admin.json

```
@@ -343,6 +343,7 @@
     "data.configDir": "Configuration folder: ",
     "data.dataDir": "Data directory: ",
     "data.formatterPath": "Formatter directory: ",
+    "data.indexConfigDir": "Index folder: ",
```

```suggestion
    "data.indexConfigDir": "Index configuration folder: ",
```
would be more appropriate. The index folder is where Elasticsearch is.